js/templates/modals/baseModal.html

```diff
@@ -1,4 +1,22 @@
+<div class="popInMessageHolder js-popInMessages"></div>
 <div class="<%= ob.modelContentClass %>">
-  <i class="jsModalClose <%= ob.closeButtonClass %> <% !ob.showCloseButton && print('hide') %>"></i>
+  <div class="topControls pureFlex spread js-closeClickTarget js-btcTicker">
```

Comment:
it's not meant as finished. I plan to find a clean way to remove the top control duplication. the current sketch is not ideal and it was quicker than injecting custom markup into top control. I can revert it back. 

and I also think factoring it out is the right move. because we can already see the top control is overloaded for many tasks. it's useful. the number of standard and custom things we want to put into there may continue to grow. it works to have a standard and robust way to style that and organise that to permit both common and custom items in the top control in a clean way with minimal duplication.